### ISSUE TITLE
Migrate to v2 embedding

### ISSUE THREAD

**Issue by danielnordh** [kikuchy/plain_notification_token#7]
This package is currently the only one we use that is not using v2 embedding.
It builds with the following warning:

```
The plugin `plain_notification_token` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will
remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```

Would be awesome to get it migrated 💯

**Comment by Vov4yk** [comment]
Faced the same issue. As a result I've made own solution. You can check if it would work for you. https://github.com/kikuchy/plain_notification_token/pull/8

**Comment by zionnite** [comment]
any progress on this so far???

**Comment by kikuchy** [comment]
Resolved on #8 . Thank you for your contribution!
